documentation/docs/20-core-concepts/40-page-options.md

```diff
@@ -78,9 +78,10 @@ For _pages_, we skirt around this problem by writing `foo/index.html` instead of
 
 If you encounter an error like 'The following routes were marked as prerenderable, but were not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't actually prerendered, because it wasn't reached by the prerendering crawler.
```

Comment:
updating this to avoid having "because" in the same sentence twice
```suggestion
If you encounter an error like 'The following routes were marked as prerenderable, but were not prerendered' it's because the route in question (or a parent layout, if it's a page) has `export const prerender = true` but the page wasn't reached by the prerendering crawler and thus wasn't prerendered.
```